solutions/usaco-834.mdx

```diff
@@ -0,0 +1,71 @@
+---
+id: usaco-1037
+source: USACO Silver 2018 US Open
+title: Out of Sorts
+author: Brad Ma
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_sort_silver_open18.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N \log N)$
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.util.*;
+import java.io.*;
+
+public class OutOfSorts {
+
+	// hold entry values
```

Comment:
what's an entry value? more specificity would be nice--mention how you're using this to track original index after array has been sorted